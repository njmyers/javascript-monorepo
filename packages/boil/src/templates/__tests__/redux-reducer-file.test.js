import reduxReducerFile from "../redux-reducer-file";

describe("it works", () => {
  test("it is a function ", () => {
    expect(typeof reduxReducerFile).toBe("function");
  });

  test("it runs without crashing", () => {
    expect(reduxReducerFile("test-component")).toMatchSnapshot();
  });
});
