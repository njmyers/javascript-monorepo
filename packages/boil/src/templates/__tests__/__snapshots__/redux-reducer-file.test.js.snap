// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it works it runs without crashing 1`] = `
"
import type { Action } from './test-component-actions'

export type State = {
  +key: string,
}

export const initialState = {
  key: 'value'
}

const testComponentReducer = (state: State = initialState, action: Action) => {
  switch(action.type) {
    default:
      return state;
  }
}

export default testComponentReducer
"
`;
